shop.go

```diff
@@ -97,3 +99,8 @@ func (shop Shop) Products() shopify.ProductRepository {
 func (shop Shop) InventoryLevels() shopify.InventoryLevelRepository {
 	return shop.inventoryLevels
 }
+
+// Collections returns an http implementation of a shopify collection repository
```

Comment:
```suggestion
// Collections returns an HTTP implementation of a Shopify collection repository
```

Reply:
this comment is following the rest of the shop methods shall I change those as well then?

Reply:
yes please